docs/admin/saml2-shibboleth.rst

```diff
@@ -96,22 +105,31 @@ The SAML2 configuration of Artemis could look like this:
         email-pattern: '{mail}'
         registration-number-pattern: '{matriculationNumber}'
         lang-key-pattern: 'de'
-        identity-providers:
-            - metadata: https://www.aai.dfn.de/fileadmin/metadata/dfn-aai-local-148-metadata.xml
-              registration-id: REGISTRATION_ID_HERE
-              entity-id: ENTITY_ID_HERE
-              cert-file: /opt/artemis/config/shibboleth-sp-cert.crt
-              key-file: /opt/artemis/config/shibboleth-sp-key.pem
-    info.saml2.button-label: 'Shibboleth Login'
-    info.saml2.enable-password: false
-    info.saml2.password-login-disabled: true
-    info.saml2.identity-provider-name: 'Shibboleth Account'
+    info:
+        saml2:
+            button-label: 'Shibboleth Login'
+            enable-password: false
+            password-login-disabled: true
+            identity-provider-name: 'Shibboleth Account'
+    spring:
```

Comment:
Everything above this line in the code snippet has indentation of 3, everything below 2.

Reply:
oops :) .. i'll fix that :D